src/help/zaphelp/contents/start/concepts/structparams.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
+<HTML>
+<HEAD>
+<META HTTP-EQUIV="Content-Type" CONTENT="text/html; charset=utf-8">
+<TITLE>
+Structural Parameters
+</TITLE>
+</HEAD>
+<BODY BGCOLOR="#ffffff">
+<H1>Structural Parameters</H1>
+<p>
+Structural parameters are a type of <a href="structmods.html">Structural Modifier</a>
+which identify parameters that represent application structure instead of user data.<br>
+<p>
+In 'traditional' web applications the structure of the application is typically defined by the URL paths
+and the data is contained in the URL parameters and POST data.<br>
+URLs like:
+<ul>
+<li>https://www.example.com/app/aaa?ddd=eee
+<li>https://www.example.com/app/aaa?ddd=fff
+<li>https://www.example.com/app/aaa?ddd=ggg
+</ul>
+are represented in the <a href="../../ui/tabs/sites.html">Sites tab</a> as one 'node' in the tree:
+<ul>
+<li>Sites
+	<ul>
+	<li>https://www.example.com
+		<ul>
+		<li>app
+			<ul>
+			<li>GET:aaa(ddd)
+			</ul>
+		</ul>
+	</ul>
+</ul>
+The Sites tree is very important as it reflects ZAP's understanding of the application structure.<br>
+If it is not a good representation of the structure then ZAP will not be able to attack the application effectively.<br>
+<br>
+In 'single' page applications a parameter is used to indicate the logical 'page':
+<ul>
+<li>https://www.example.com/app/aaa?page=p1&ddd=eee
```

Comment:
The ampersand should be replaced with corresponding HTML entity (same comment for other ampersands).